content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Jellyfin 10.5"
+subtitle = "A step closer to the best experience with your media"
+author = "Fernando"
+githubusername = "ferferga"
+date = "2020-03-06"
+justify = "center"
++++
+
+Hello Jellyfin users out there! Welcome back to our blog!
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!
+
+We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
+We hope you will forgive us (*send everyone who contributed a cake if you do, please!*) after trying it!
+
+# Key Features
+
+## Let your content shine like never before
+
+If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:
+
+#### Jellyfin 10.4.3
+
+<img src="/images/10.5-release/old.png" name="Old item details page" />
+
+#### Jellyfin 10.5
+
+<img src="/images/10.5-release/new.png" name="New item details page" />
+
+*With 10.5, we wanted to reimagine Jellyfin's Web interface*
+
+Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was the jungle when
+I took that screenshot of the *Jungle's Book* wallpaper. Do you feel it like me?
+
+Notice also that the header of the UI has been reworked a little bit in a *slightly* lighter color (instead of the plain color used by the theme)
+
+Shoutout to all the Web team for their invaluable contributions in mking this happen, specially to *[Julien](https://github.com/mrtimscampi)* and 
+*[grafixeyehero](https://github.com/grafixeyehero)*.
```

Comment:
```suggestion
Jellyfin 10.5.0 is the start of a major user interface rework that will be spread over the coming releases.

This release starts with a complete overhaul of the details screen, where posters are now as relevant as the description of your item and backdrops are front and center.

The header also has a tighter layout and a lighter color in our default Dark theme, to make the page less monotone.

We'd like to thank every contributor to the web client for their invaluable contributions in making this happen, especially to *[MrTimscampi](https://github.com/mrtimscampi)* and *[grafixeyehero](https://github.com/grafixeyehero)*.
```

Rewording of a few paragraphs to be less conversational and familiar. Also use my nickname, since that's what I appear as on all the community platforms (It's easier for people to make the link, even if I don't care about using my real name).

Reply:
I feel that removing most of that is maybe too much, I think we need to give more selling points and an appeal to the updates, so people *want* to upgrade/switch, not *need to* (see #21)

I will make some of your proposed changes in this section in the wording of some elements, so take a look at the commit that I'm going to push.